wpimath/src/main/native/include/frc/controller/ArmFeedforward.h

```diff
@@ -76,6 +80,53 @@ class WPILIB_DLLEXPORT ArmFeedforward {
            kV * velocity + kA * acceleration;
   }
 
+  /**
+   * Calculates the feedforward from the gains and setpoints.
+   *
+   * @param currentAngle        The current angle, in radians. This
+   * angle should be measured from the horizontal (i.e. if the provided angle is
+   * 0, the arm should be parallel to the floor). If your encoder does not
+   * follow this convention, an offset should be added.
+   * @param nextAngle        The next angle, in radians. This
+   * angle should be measured from the horizontal (i.e. if the provided angle is
+   * 0, the arm should be parallel to the floor). If your encoder does not
+   * follow this convention, an offset should be added.
+   * @param currentVelocity The current velocity setpoint, in radians per
+   *                        second.
+   * @param nextVelocity    The next velocity setpoint, in radians per second.
+   * @param dt              Time between velocity setpoints in seconds.
+   * @return The computed feedforward, in volts.
+   */
+  units::volt_t Calculate(units::unit_t<Angle> currentAngle,
+                          units::unit_t<Angle> nextAngle,
+                          units::unit_t<Velocity> currentVelocity,
+                          units::unit_t<Velocity> nextVelocity,
+                          units::second_t dt) const {
+    Vectord<2> r{currentAngle.value(), currentVelocity.value()};
+    Vectord<2> nextR{nextAngle.value(), nextVelocity.value()};
+
+    sleipnir::OptimizationProblem problem;
```

Comment:
I'd potentially be concerned with running a Sleipnir optimisation problem on the roboRIO especially a roboRIO 1
Might be ok here as its a relatively simple problem though

Reply:
By using doubles for everything, you're telling Sleipnir to minimize a constant, which does nothing.

Reply:
BTW Just to be clear I am okay to wait on this being until @calcmogul et al. does a PR on Sleipnir separately.  I just needed it to get the ball rolling on a solution (and feedback) to the ArmFeedforward calculate solution.  Not trying to steal anyones thunder.  I apologize if it seemed like I was.

Reply:
> By using doubles for everything, you're telling Sleipnir to minimize a constant, which does nothing.

Is this a problem on my end or were you responding to @spacey-sooty .  If on me.  How do I fix?